.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
+                                        def postCommitPhrase = '') {
```

Comment:
Used an underscore instead (keeping in mind upcoming change to move to all underscores) to avoid name collision with triggerPhrase() method.